.gitmodules

```diff
@@ -49,7 +49,7 @@
 	url = https://github.com/rust-lang/llvm
 [submodule "src/stdsimd"]
 	path = src/stdsimd
-	url = https://github.com/rust-lang-nursery/stdsimd
+	url = https://github.com/alexcrichton/stdsimd
```

Comment:
Why doesn't this still use rust-lang-nursery?

Reply:
Because this isn't ready to land yet, it's waiting on FCP to finish.